02.extensions/zooitempro/01.basics/06.faq/docs.md

```diff
@@ -0,0 +1,8 @@
+---
+title: F.A.Q.
+taxonomy:
+    category: docs
+---
+
+* **Does ZOOitem Pro filtering support Date Pro element?** <br /> ZOOitem Pro supports single not repeatable date events.
```

Comment:
change it to "How to filter items based on DatePro element? Currently only single, not repeatable, dates are supported.